readme.md

```diff
@@ -73,8 +73,28 @@ npm run setup
 
 4. Build & launch <a target="_blank" href="https://singlelink.co">Singlelink</a>
 
+Build all the modules
+
+```bash
+npm run build
+```
+
+Start the API in one terminal:
```

Comment:
Is there one comand we can wrap all of the servers into to make things easier?

Reply:
@jimmybisenius was actually working on a setup script (setup.js) that would work with npm, but it hasn't been completed afaik. That would allow for some commands that could run on the root-level package.